app/views/users/index.html.slim

```diff
@@ -41,4 +41,7 @@ table.users.table.table-striped.table-bordered.table-condensed.table-hover
 
 p.pagination-info
   = page_entries_info @users
-  = paginate @users, theme: 'twitter-bootstrap-3', pagination_class: 'pagination-sm'
\ No newline at end of file
+  = paginate @users, theme: 'twitter-bootstrap-3', pagination_class: 'pagination-sm'
+  / gets this @users from users index
+    .unscope(:order, :select, :where) == User.all
```

Comment:
I intented to leave the comments untill after Carsten had a chance to look at it, just to make my own life easier if Carsten asks me to do something else with it. I will not ship with this comments. 

Can I leave them untill I myself don't need them anymore? It saves me lots of time. Or do you consider it bad practice to have this kind of comments in PR's at all?